src/Listbox.tsx

```diff
@@ -101,21 +97,34 @@ export class Listbox extends Component<Props, State> {
     })
   }
 
+  getFocusValue = (): string | null => {
+    const activeValue = this.props.value
+    // Set active value to be the first option
+    // in the list if no item is selected.
+    // https://www.w3.org/TR/wai-aria-practices/#listbox_kbd_interaction
+    if (!activeValue){ return this.state.values[0] }
+    return Array.isArray(activeValue) ? activeValue[0] : activeValue
```

Comment:
Yep sounds good

Reply:
What do you think about having a `values` prop for the mutliselect version. That way we don't have to check if it's an array or not everywhere.

We could also error early in the `Listbox` component to prevent people from using the wrong props
```
if (this.props.multiselect && this.props.value) {
    throw new Error(`Use the "values" prop for multiselect`)
}

if (!this.props.multiselect && this.props.values) {
   throw new Error(`Use the "value" prop for non-multiselect`)
}
```

Reply:
I think I figured out a way to do this without errors in typescript with a union type
```
interface SharedProps {
  children: React.ReactNode,
  className?: string,
  onChange: (value: string | string[] | null) => void
}

interface SingleProps extends SharedProps {
  multiselect: false;
  value: string | null,
}

interface MultiProps extends SharedProps {
  multiselect: true;
  values: string[];
}
```

Reply:
I've pushed some changes to do it the union type way above ^

Keen to hear your feedback! Thanks

Reply:
I would additionally add-in `never`.

```
interface SharedProps {
  children: React.ReactNode
  className?: string
  onChange: (value: string | string[] | null) => void
}

interface SingleProps extends SharedProps {
  multiselect: false
  value: string | null
  values: never
}

interface MultiProps extends SharedProps {
  multiselect: true
  values: string[]
  value: never
}

type Props = SingleProps | MultiProps
```

Reply:
Sounds good. I added that in: https://github.com/mitchbne/react-listbox/pull/5/commits/f9f2f0caaefa13f092950665acc16524b4d6186f

Reply:
Actually this seems to create a TS error. Looking into why.

Reply:
I removed it again. Seems it was expecting the property to be defined even though it was never. If you figure it out let me know. Thanks!